spinta/types/backref/link.py

```diff
@@ -95,7 +95,10 @@ def link(context: Context, dtype: BackRef) -> None:
         raise OneToManyBackRefNotSupported(dtype)
 
     # relationship needs to add unique
-    dtype.refprop.dtype.unique = True
+    # fixme at least in some cases (one to many relationship, for example) this is setting unique incorrectly
```

Comment:
```suggestion
    # FIXME: at least in some cases (one to many relationship, for example) this is setting unique incorrectly
```

Žiūrėti: https://peps.python.org/pep-0350/